### DISCUSSION TITLE
Welcome to my_website Discussions!

### DISCUSSION THREAD

**Discussion by alves-Moises** [alves-Moises/my_website#4] [Show and tell]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vinicius-rech** [comment]
teste

**Reply by alves-Moises** [reply]
testing
## Testing.. 
# Testing!

**Comment by vinicius-rech** [comment]
teste2

**Comment by vinicius-rech** [comment]
qwe qw ewq qwe
